src/index.ts

```diff
@@ -1,158 +1,67 @@
-  p0_001: number;
-  p0_01: number;
-  p0_1: number;
-  p1: number;
-  p2_5: number;
-  p10: number;
-  p25: number;
-  p50: number;
-  p75: number;
-  p90: number;
-  p97_5: number;
-  p99: number;
-  p99_9: number;
-  p99_99: number;
-  p99_999: number;
-}
-/* eslint-enable camelcase */
-
-function createHistogramSummary (histogram: Histogram): HistogramSummary {
-  const { mean, stddev, min, max } = histogram;
-
-  return {
-    average: mean / 1000,
-    mean: mean / 1000,
-    stddev,
-    min: min / 1000,
-    max: max / 1000,
-    p0_001: histogram.percentile(0.001) / 1000,
-    p0_01: histogram.percentile(0.01) / 1000,
-    p0_1: histogram.percentile(0.1) / 1000,
-    p1: histogram.percentile(1) / 1000,
-    p2_5: histogram.percentile(2.5) / 1000,
-    p10: histogram.percentile(10) / 1000,
-    p25: histogram.percentile(25) / 1000,
-    p50: histogram.percentile(50) / 1000,
-    p75: histogram.percentile(75) / 1000,
-    p90: histogram.percentile(90) / 1000,
-    p97_5: histogram.percentile(97.5) / 1000,
-    p99: histogram.percentile(99) / 1000,
-    p99_9: histogram.percentile(99.9) / 1000,
-    p99_99: histogram.percentile(99.99) / 1000,
-    p99_999: histogram.percentile(99.999) / 1000
-  };
-}
-
-function toHistogramIntegerNano (milliseconds: number): number {
-  return Math.max(1, Math.trunc(milliseconds * 1000));
-}
-
-interface AbortSignalEventTargetAddOptions {
-  once : boolean;
-};
-
-interface AbortSignalEventTarget {
-  addEventListener : (
-    name : 'abort',
-    listener : () => void,
-    options? : AbortSignalEventTargetAddOptions) => void;
-  removeEventListener : (
-    name : 'abort',
-    listener : () => void) => void;
-  aborted? : boolean;
-  reason?: unknown;
-}
-interface AbortSignalEventEmitter {
-  off : (name : 'abort', listener : () => void) => void;
-  once : (name : 'abort', listener : () => void) => void;
-}
-type AbortSignalAny = AbortSignalEventTarget | AbortSignalEventEmitter;
-function onabort (abortSignal : AbortSignalAny, listener : () => void) {
-  if ('addEventListener' in abortSignal) {
-    abortSignal.addEventListener('abort', listener, { once: true });
-  } else {
-    abortSignal.once('abort', listener);
-  }
-}
-
-class AbortError extends Error {
-  constructor (reason?: AbortSignalEventTarget['reason']) {
-    // TS does not recognizes the cause clause
-    // @ts-expect-error
-    super('The task has been aborted', { cause: reason });
-  }
-
-  get name () { return 'AbortError'; }
-}
-
-type ResourceLimits = Worker extends {
-  resourceLimits? : infer T;
-} ? T : {};
-type EnvSpecifier = typeof Worker extends {
-  new (filename : never, options?: { env: infer T }) : Worker;
-} ? T : never;
 
-export class ArrayTaskQueue implements TaskQueue {
-  tasks : Task[] = [];
-
-  get size () { return this.tasks.length; }
-
-  shift () : Task | null {
-    return this.tasks.shift() as Task;
-  }
-
-  push (task : Task) : void {
-    this.tasks.push(task);
-  }
+const { version } = JSON.parse(
```

Comment:
@metcoder95 hi, this is currently breaking things for us. 

Is this `version` extraction required? can we pass in a custom path or version in options instead?

This breaks flexibility in a number of ways, like using esbuild and bundling.

Reply:
Let me revert, this is causing too many problems. Will open a PR and publish a fix soon